pkg/kubelet/config/common.go

```diff
@@ -73,6 +73,21 @@ func applyDefaults(pod *api.Pod, source string, isFile bool, nodeName types.Node
 	// The generated UID is the hash of the file.
 	pod.Annotations[kubetypes.ConfigHashAnnotationKey] = string(pod.UID)
 
+	if isFile {
+		// Applying the default Taint tolerations to static pods,
+		// so they are not evicted when there are node problems.
```

Comment:
Sorry, to clarify -- what I wrote above is about the tolerations, not the taints. But that wasn't your question.

But yes, we will have to add the taints here when there are new ones. We can probably create some kind of const array that contains all of them, so they only have to be updated in one place, or something like that.